dev/NumberBox/NumberBox.xaml

```diff
@@ -20,6 +20,14 @@
                 <ControlTemplate TargetType="local:NumberBox">
                     <Grid>
                         <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="DisabledStates">
+                                <VisualState x:Name="Enabled"/>
```

Comment:
Convention is to call this "Normal" instead I believe

Reply:
Rob is right, at least while there are a part of the CommonStates group the default stat is conventionally called Normal

---
In reply to: [462024533](https://github.com/microsoft/microsoft-ui-xaml/pull/3005#discussion_r462024533) [](ancestors = 462024533)

Reply:
Done in the newest commit.